README.md

```diff
@@ -30,14 +30,32 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+The biggest difference between two is that .forEach executes a provided function on each of the array elements mutating the original array. On the other hand, .map does not mutate the original array but rather executes a provided function on each array element returning the new results in a new array (while leaving the original array intact.)
+
 2. What is the difference between a function and a method?
 
+A function is a piece of code that can be called by its name and can be passed data as a parameter/s. A method is an action that can be performed on an object.
+
```

Comment:
Don't forget about anonymous functions. I would this answer as:
`A method is a function that belongs to an object, and if it is part of a class or constructor function, it can be inherited.`